### PR TITLE
#633 Explain requirements for execution plane services in the dev docs

### PR DESCRIPTION
**Pull request by johannes-b**
**To Do (double-check)**
- [x] Events shown in the docs must refer to: "specversion": "1.0"
- [x] Bug-fix with keywords (used for "See also" section)

### CODE REVIEW

File: content/docs/0.7.x/operate/backup_and_restore/index.md [hunk 1]
```diff
@@ -5,14 +5,19 @@ weight: 20
...
+backup the data within the *Configuration Service*, the *MongoDB*, and credentials to your projects' Git upstream repos (if you have configured those).
+The following sections describe how to back up that data and how to restore it.
 
-## Backup Configuration Service
+**Note:** These instructions do not cover backing up any Helm- or Istio-related configurations.
 
-First, create a local backup directory containing all data within the git repos of the configuration-service. To do so, please execute the following command:
+## Backup Keptn
+
+This section describes how to backup your Keptn and store it on your local maschine.
```
Comment (bacherfl) [inline content/docs/0.7.x/operate/backup_and_restore/index.md:16]: typo: "maschine"

File: content/docs/0.8.x/integrations/action_provider/index.md [hunk 2]
```diff
@@ -5,20 +5,32 @@ weight: 10
...
+* has a **subscription** to an event (i.e., `sh.keptn.event.action.triggered`)
+* sends a **started event** to inform Keptn about receiving the event and acting on it
+* processes functionality and can therefore leverage additional tools, e.g., through their REST interface
+* sends a **finished event** to inform Keptn about its execution status and the result 
+
+### Subscription to Keptn event
+
+An *action-provider* starts working, when receiving a Keptn CloudEvent of type:
+
+-  [sh.keptn.event.action.triggered](https://github.com/keptn/spec/blob/0.1.6/cloudevents.md#action-triggered)
```
Comment (bacherfl) [inline content/docs/0.8.x/integrations/action_provider/index.md:25]: we will need to update the link to the spec once we have released the spec containing the [Keptn 0.8 Events](https://github.com/keptn/spec/pull/45)

File: content/docs/0.8.x/integrations/action_provider/index.md [hunk 3]
```diff
@@ -43,60 +55,60 @@ The [Keptn CloudEvents](#cloudevents) an action-provider has to subscribe to is:
...
+```json
+{
+  "type": "sh.keptn.event.action.finished",
+  "specversion": "1.0",
+  "source": "https://github.com/keptn/keptn/unleash-service",
+  "id": "ggb878d3-03c0-4e8f-bc3f-454bc1b3d888",
+  "time": "2019-06-07T07:02:15.64489Z",
+  "contenttype": "application/json",
+  "shkeptncontext": "08735340-6f9e-4b32-97ff-3b6c292bc509",
+  "triggeredid": "2b878d3-03c0-4e8f-bc3f-454bc1b3d79d",
```
Comment (bacherfl) [inline content/docs/0.8.x/integrations/action_provider/index.md:88]: let's add a short description on how to set the triggeredid

File: content/docs/0.8.x/integrations/sli_provider/index.md [hunk 4]
```diff
@@ -18,69 +18,133 @@ indicators:
...
+* has a **subscription** to an event (i.e., `sh.keptn.event.get-sli.triggered`)
+* sends a **started event** to inform Keptn about receiving the event and acting on it
+* processes functionality and can therefore leverage additional tools, e.g., through their REST interface
+* sends a **finished event** to inform Keptn about its execution status and the result
+
+### Subscription to Keptn event
+
+An *SLI-provider* starts working, when receiving a Keptn CloudEvent of type:
+
+- [sh.keptn.event.get-sli.triggered](https://github.com/keptn/spec/blob/0.1.6/cloudevents.md#get-sli)
```
Comment (bacherfl) [inline content/docs/0.8.x/integrations/sli_provider/index.md:38]: we will need to update the link to the spec once we have released the spec containing the Keptn 0.8 Events

File: content/docs/0.8.x/integrations/sli_provider/index.md [hunk 5]
```diff
@@ -18,69 +18,133 @@ indicators:
...
+```json
+{
+  "type": "sh.keptn.event.get-sli.finished",
+  "specversion": "1.0",
+  "source": "https://github.com/keptn/keptn/dynatrace-sli-service",
+  "id": "ggb878d3-03c0-4e8f-bc3f-454bc1b3d888",
+  "time": "2019-06-07T07:02:15.64489Z",
+  "contenttype": "application/json",
+  "shkeptncontext": "08735340-6f9e-4b32-97ff-3b6c292bc509",
+  "triggeredid": "2b878d3-03c0-4e8f-bc3f-454bc1b3d79d",
```
Comment (bacherfl) [inline content/docs/0.8.x/integrations/sli_provider/index.md:112]: let's add a short description on how to set the triggeredid

File: content/docs/0.8.x/operate/backup_and_restore/index.md [hunk 6]
```diff
@@ -5,14 +5,19 @@ weight: 20
...
+backup the data within the *Configuration Service*, the *MongoDB*, and credentials to your projects' Git upstream repos (if you have configured those).
+The following sections describe how to back up that data and how to restore it.
 
-## Backup Configuration Service
+**Note:** These instructions do not cover backing up any Helm- or Istio-related configurations.
 
-* First, create a local backup directory containing all data within the git repos of the configuration-service. To do so, please execute the following command:
+## Backup Keptn
+
+This section describes how to backup your Keptn and store it on your local maschine.
```
Comment (bacherfl) [inline content/docs/0.8.x/operate/backup_and_restore/index.md:16]: typo: maschine

**Review by bacherfl — CHANGES_REQUESTED** [review]

File: content/docs/0.7.x/operate/backup_and_restore/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/0.7.x/operate/backup_and_restore/index.md:16]: fixed

File: content/docs/0.8.x/operate/backup_and_restore/index.md (hunk 6 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/backup_and_restore/index.md:16]: fixed

File: content/docs/0.8.x/integrations/action_provider/index.md (hunk 2 above)
Comment (johannes-b) [reply content/docs/0.8.x/integrations/action_provider/index.md:25]: True. Therefore we have the issue (https://github.com/keptn/keptn.github.io/issues/667) that addresses this when the spec is online.

File: content/docs/0.8.x/integrations/sli_provider/index.md (hunk 4 above)
Comment (johannes-b) [reply content/docs/0.8.x/integrations/sli_provider/index.md:38]: True. Therefore we have the issue (https://github.com/keptn/keptn.github.io/issues/667) that addresses this when the spec is online.

File: content/docs/0.8.x/integrations/action_provider/index.md (hunk 3 above)
Comment (johannes-b) [reply content/docs/0.8.x/integrations/action_provider/index.md]: added a description now

File: content/docs/0.8.x/integrations/sli_provider/index.md (hunk 5 above)
Comment (johannes-b) [reply content/docs/0.8.x/integrations/sli_provider/index.md]: added a description

**Review by bacherfl — APPROVED** [review]

**Pull request merged**